packages/core/src/html/includePanelProcessor.js

```diff
@@ -58,16 +57,26 @@ function _getBoilerplateFilePath(node, filePath, config) {
  * Retrieves several flags and file paths from the src attribute specified in the element.
  */
 function _getSrcFlagsAndFilePaths(element, config) {
-  const isUrl = urlUtil.isUrl(element.attribs.src);
+  let isUrl = urlUtil.isUrl(element.attribs.src);
 
   // We do this even if the src is not a url to get the hash, if any
-  const includeSrc = url.parse(element.attribs.src);
+  let newURL = `${config.baseUrl ? config.baseUrl : 'http://127.0.0.1:8080'}${element.attribs.src}`;
+
+  // Check again if newURL is a full valid URL before creating URL object
+  // Workaround for running tests with missing protocol, hostname and port
+  isUrl = urlUtil.isUrl(newURL);
```

Comment:
`src` that are not valid urls (e.g `/userGuide/addingPages.md` ) will be processed in the `else` branch below to retrieve the file path of the `md` file on the system (e.g `C:\markbind\docs\userGuide\addingPages.md`). The `newUrl` will result in the above `src` having a valid url and hence not getting the correct file path.

Reply:
Thanks for clarifying this jonah! I missed out on this. Will make the changes i.e. Shift all the `URL` related codes into the `else` branch instead.